test/OpenTelemetry.Exporter.Zipkin.Tests/Implementation/ZipkinActivityConversionTest.cs

```diff
@@ -125,5 +126,110 @@ public void ToZipkinSpan_Status_ErrorFlagTest(StatusCode expectedStatusCode, str
                 Assert.DoesNotContain(zipkinSpan.Tags, t => t.Key == "error");
             }
         }
+
+        [Theory]
+        [InlineData(ActivityStatusCode.Unset)]
+        [InlineData(ActivityStatusCode.Ok)]
+        [InlineData(ActivityStatusCode.Error)]
+        public void ToZipkinSpan_Activity_Status_And_StatusDescription_is_Set(ActivityStatusCode expectedStatusCode)
+        {
+            // Arrange.
+            const string description = "Description when ActivityStatusCode is Error.";
+            var activity = ZipkinExporterTests.CreateTestActivity();
+            activity.SetStatus(expectedStatusCode, description);
+
+            // Act.
+            var zipkinSpan = activity.ToZipkinSpan(DefaultZipkinEndpoint);
+
+            // Assert.
+            if (expectedStatusCode == ActivityStatusCode.Unset)
+            {
+                Assert.DoesNotContain(zipkinSpan.Tags, t => t.Key == SpanAttributeConstants.StatusCodeKey);
+            }
+            else
+            {
+                Assert.Equal(
+                    StatusHelper.GetTagValueForActivityStatusCode(expectedStatusCode),
+                    zipkinSpan.Tags.FirstOrDefault(t => t.Key == SpanAttributeConstants.StatusCodeKey).Value);
+            }
+
+            if (expectedStatusCode == ActivityStatusCode.Error)
+            {
+                Assert.Contains(
+                    zipkinSpan.Tags, t =>
+                    t.Key == ZipkinActivityConversionExtensions.ZipkinErrorFlagTagName &&
+                    (string)t.Value == description);
+            }
+            else
+            {
+                Assert.DoesNotContain(
+                    zipkinSpan.Tags, t =>
+                    t.Key == ZipkinActivityConversionExtensions.ZipkinErrorFlagTagName);
+            }
+        }
+
+        [Fact]
+        public void ActivityStatus_Takes_precedence_Over_Status_Tags_ActivityStatusCodeIsOk()
+        {
+            // Arrange.
+            var activity = ZipkinExporterTests.CreateTestActivity();
+            activity.SetStatus(ActivityStatusCode.Ok);
+            activity.SetTag(
```

Comment:
I think there is an issue with this test. Setting the status code tag value to something outside of pre-defined status values will result in value not being added to exported tags.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/58f13ba7e2a7f31c82db67c8c4999d7f90a78ce7/src/OpenTelemetry.Api/Internal/StatusHelper.cs#L47

Reply:
Fixed.